Sources/Benchmark/OperatingSystemStats/OperatingSystemStatsProducer+Darwin.swift

```diff
@@ -13,7 +13,7 @@
     import Darwin
     import Dispatch
 
-    public class OperatingSystemStatsProducer {
+    internal class OperatingSystemStatsProducer {
```

Comment:
'internal' is a default access level, isn't it?

Reply:
Yeah, it is, thanks.